test/turbopack-tests-manifest.js

```diff
@@ -25,32 +25,20 @@ const enabledTests = [
   'test/development/jsconfig-path-reloading/index.test.ts',
   'test/development/middleware-warnings/index.test.ts',
   'test/development/next-font/deprecated-package.test.ts',
-  'test/development/project-directory-with-styled.jsx',
```

Comment:
weird to say, but these tests were `reliably` flaky enough so I consider as failing test for now and removed from the manifest.